db/ddl.sql

```diff
@@ -78,5 +78,6 @@ CREATE TABLE `litle_responses` (
   `updated_at` datetime NOT NULL,
   `kb_account_id` varchar(255) DEFAULT NULL,
   `kb_tenant_id` varchar(255) DEFAULT NULL,
+  `approved_amount` int(11) DEFAULT NULL,
```

Comment:
This is to approve partial authorizations, if a full amount authorization cannot be secured.